api/specs/webserver/v0/components/schemas/project-v0.0.1.json

```diff
@@ -155,6 +155,9 @@
                         "store": {
                           "type": ["string", "integer"]
                         },
+                        "dataset": {
+                          "type": "string"
+                        },
```

Comment:
not sure I get why you need this?

Reply:
Cause the fetching of metadata from storage is done by providing location (store), dataset and path